wordlist

```diff
@@ -155,6 +155,7 @@ HLLs
 HMAC-SHA256
 HVM
 HW
+Hardcoded
```

Comment:
I don't think we need the uppercase versions of these words. That's for names which are always spelled in uppercase. So please keep only the lowercase versions.

Reply:
We might if they appear at a beginning of the sentence.

Reply:
I added lowercase entries as well.

Reply:
Are you sure? I believe lowercase wordlist words are allowed with both lowercase and uppercase (in the beginning of a sentence) while uppercase words in the wordlist are only allowed with uppercase.

Reply:
I don't think it hurts to leave them in. I will check to see whether we need them, as I did see other similar uppercase words there. That said, is it ok to approve this PR?

Reply:
Fine, it doesn't hurt. :-)